docs/extras/styling.md

```diff
@@ -150,3 +150,79 @@ The `display_port_labels` render parameter allows you to display the port labels
         <img src="../../assets/images/ContextDiagram of Hierarchy display_port_labels.svg" width="1000000">
         <figcaption>Context diagram of Hierarchy LogicalComponenet with type [LAB] display_port_labels</figcaption>
     </figure>
+
+# PVMT Styling
+
+The `pvmt_styling` render parameter allows you to dynamically style diagram elements based on PVMT (Property Value Management Tool) properties attached to model elements in Capella. This enables custom coloring and styling based on model metadata.
+
+!!! info "PVMT Integration"
+
+    This feature requires PVMT properties to be defined in your Capella model. The styling system reads color values from specific PVMT properties and applies them to diagram elements.
+
+## Configuration
+
+The `pvmt_styling` parameter accepts a dictionary with the following structure:
+
+``` yaml
+pvmt_styling:
+  value_groups:
+    - "Test.Kind.Color"  # List of PVMT value group names
+  children_coloring: false  # Whether to apply styling to child elements
```

Comment:
If `children_coloring` should be left false, it would be very convenient if one could skip the additional `value_groups` sub-element and flatten the passed in value to just the list-of-strings (or even single string) for the PVMT group(s):

```py
dg.render(..., pvmt_styling={"value_groups": ["Test.Kind.Color"]})
dg.render(..., pvmt_styling=["Test.Kind.Color"])
dg.render(..., pvmt_styling="Test.Kind.Color")
```

It'd be slightly more overhead for decoding the passed-in value, but especially the first step does quite nicely simplify the interface, so IMO the small amount of overhead would be worth it.